### PR TITLE
docs: adding big bang app-arch diagram and reorg of the repo to accom…

### PR DESCRIPTION
**Pull request by MxNxPx**
if it makes sense to live here, would like to pull in an application architecture type diagram for big bang

### CODE REVIEW

**Comment by MxNxPx** [comment]
adding @runyontr to get his thoughts as well

**Comment by RothAndrew** [comment]
Can you pull the license change out into a separate PR? Other than that I don't have any objections to this. It looks to mostly provide an alternative style of diagram which, while harder to read, is very likely more technically accurate.

**Comment by RothAndrew** [comment]
The main question is whether we want to maintain both styles or pick one and delete the other.

I have no horse in this race so I'm fine with the C4 stuff going away if it means better maintainability and the other style has more value.

**Review by RothAndrew — DISMISSED** [review]
see above comments

**Comment by MxNxPx** [comment]
> Can you pull the license change out into a separate PR? Other than that I don't have any objections to this. It looks to mostly provide an alternative style of diagram which, while harder to read, is very likely more technically accurate.

good catch, overwriting the LICENSE was my oopsie.  reverting that back to the last commit.

**Comment by MxNxPx** [comment]
> The main question is whether we want to maintain both styles or pick one and delete the other.
> 
> I have no horse in this race so I'm fine with the C4 stuff going away if it means better maintainability and the other style has more value.

i feel like both are useful points of view on the big bang architecture.  but if splitting the repo up in the PR proposed manner breaks some other documentation links or perhaps is not very useful, i am good with retracting the PR and seeing if it makes sense for the new diagram to have a different home.

@RothAndrew & @runyontr - what do you think?

**Comment by RothAndrew** [comment]
I don't know if changing to the new folder structure will break any links, but if it does it won't be the end of the world for people to just update to the new location.

File: README.md [hunk 1]
```diff
@@ -1,21 +1,7 @@
 # bigbang-architecture-diagrams
 
-This repo contains architecture diagrams for Platform One's Big Bang secure software stack. They are saved as editable SVG files that can be opened and edited on [app.diagrams.net](https://app.diagrams.net). 
+This repo contains architecture diagrams for Platform One's Big Bang secure software stack.
 
-These diagrams use the [C4 Model](https://c4model.com/), where system **C**ontexts are broken down into **C**apsules, which are broken down into **C**omponents, which are broken down into **C**ode
+* [c4](./c4)
```
Comment (RothAndrew) [inline README.md:5]:
```suggestion
* [Simple but visually appealing](./c4) -- Uses the [C4 model](https://c4model.com/)
```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,7 @@
...
-
-## Capsule Diagram - Mission app running on Big Bang (with CNAP)
-
-![Capsule Diagram - Mission app running on Big Bang (with CNAP)](capsule_bb_cnap.drawio.svg)
-
-## Capsule Diagram - CNAP
-
-![Capsule Diagram - CNAP](capsule_cnap.drawio.svg)
\ No newline at end of file
+* [app-arch](./app-arch)
```
Comment (RothAndrew) [inline README.md:7]:
```suggestion
* [Technical](./app-arch) -- Uses [d2](https://github.com/terrastruct/d2)
```

**Review by RothAndrew — CHANGES_REQUESTED** [review]

**Comment by MxNxPx** [comment]
@RothAndrew - I like your suggested changes so I went ahead and committed them to the branch.
Let me know if there is anything else we may want to change or review before merging or dropping the PR.

**Review by RothAndrew — APPROVED** [review]

**Comment by RothAndrew** [comment]
@MxNxPx is this ready to merge?

**Comment by RothAndrew** [comment]
@MxNxPx bump

**Comment by RothAndrew** [comment]
Closing as abandoned

**Pull request closed without merging**